gradoop-flink/src/test/java/org/gradoop/flink/algorithms/gelly/vertexdegrees/VertexDegreesTest.java

```diff
@@ -0,0 +1,44 @@
+package org.gradoop.flink.algorithms.gelly.vertexdegrees;
+
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.Test;
+
+public class VertexDegreesTest extends GradoopFlinkTestBase {
+
+	@Test
+	public void testByElementData() throws Exception {
+		
+		String graph = "input[" +
+		"(v0 {id:0, degree:NULL, inDegree:NULL, outDegree:NULL})" +
```

Comment:
Does this also work if they don't have the properties beforehand? If yes, I would remove them from the input graph.

Reply:
I tried it and yes it does. So I remove them now.